### PR TITLE
Update app-configuration-policies-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the steps to Type about:intunehelp in the address box. instead of clicking on about intune help as it does not show up any more(tested in 2 devices)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/apps/app-configuration-policies-overview.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ You can validate the app configuration policy using the following three methods:
 You can validate iOS/iPadOS configuration with the **Intune Diagnostic Log** for settings deployed through the managed app configuration policies. In addition to the below steps, you can access managed app logs using Microsoft Edge. For more information, see [Use Edge for iOS and Android to access managed app logs](manage-microsoft-edge.md#use-edge-for-ios-and-android-to-access-managed-app-logs).
 
 1. If not already installed on the device, download and install the **Microsoft Edge** from the App Store. For more information, see [Microsoft Intune protected apps](apps-supported-intune-apps.md).
-2. Launch the **Microsoft Edge** and select **about** > **intunehelp** from the navigation bar.
+2. Launch the **Microsoft Edge** and Type about:intunehelp in the address box.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-configuration-policies-overview.md:144]:
```suggestion
2. Launch the **Microsoft Edge** and enter **about:intunehelp** in the address box.
```

File: memdocs/intune/apps/app-configuration-policies-overview.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ Your application configuration details should match the application configuratio
 You can validate iOS/iPadOS configuration with the **Intune Diagnostic Log** on managed devices for managed app configuration.
 
 1. If not already installed on the device, download and install the **Microsoft Edge** from the App Store. For more information, see [Microsoft Intune protected apps](apps-supported-intune-apps.md).
-2. Launch **Microsoft Edge** and select **about** > **intunehelp** from the navigation bar.
+2. Launch **Microsoft Edge** and Type about:intunehelp in the address box.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-configuration-policies-overview.md:188]:
```suggestion
2. Launch **Microsoft Edge** and enter **about:intunehelp** in the address box.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger13** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please merge this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**